### PR TITLE
Create user.md to store user management content

### PR DESCRIPTION
**Pull request by ysuarez**
**GitHub Issue**: #1040

# What does this Pull Request do?

Creates user.md to store user management content

Including sections for:
- How to Add a User
- How to Create a New User Role
- How to Edit Role Permissions

# What's new?
It covers the common steps for managing users in slightly more depth than older content in docs. It mentions the existence of the Drupal role of "fedoraAdmin".

# How should this be tested?

A description of what steps someone could take to:
* Review the documented steps and images to see if there is incorrect or missing information.

# Additional Notes:
I think I need improvements to the content that mentions how the Drupal roles of "Administrator" and "fedoraAdmin" are meant to be used when creating or editing a Drupal user.

Here is what I wrote so far...
>In the *Roles* section, click on the roles that the new user will need.
>  - Click on "Administrator" if the new user will need to be able to run adminsitrative tasks in Drupal or Islandora 8.
>  - Click on "fedoraAdmin" if the new user will need to be able to run certain tasks in Fedora. This is required by the WebAC authentication so Islandora 8 and Fedora can communicate.

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

File: docs/user-documentation/users.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Managing Users
+
+## How to Add a User
+1. Log in to Drupal under admin credentials.
```
Comment (librarychik) [inline docs/user-documentation/users.md]: Suggested change:
`Log in to Drupal under admin credentials.`
To:
Log into Drupal.  You will need the Administrator role to Manage Users.

File: docs/user-documentation/users.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+1. Click the **Manage** tab from the top toolbar.
+1. Select the **People** tab from the resulting menu to bring up the *People* page.
+![Alt text](../assets/users_people.png "People page")
+1. Click the blue **Add user** button to bring up the *Add user* page.
+1. Enter all required fields, as indicated by red asterisks, in the *Add user* page.
+1. In the *Roles* section, click on the roles that the new user will need.
```
Comment (librarychik) [inline docs/user-documentation/users.md]: Suggested addition:
In the *Contact Settings* section.  Note: the Personal contact form is selected by default.

File: docs/user-documentation/users.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - Click on "Administrator" if the new user will need to be able to run adminsitrative tasks in Drupal or Islandora 8.
+  - Click on "fedoraAdmin" if the new user will need to be able to run certain tasks in Fedora. This is required by the WebAC authentication so Islandora 8 and Fedora can communicate.
```
Comment (librarychik) [inline docs/user-documentation/users.md]: Suggested addition:
In the *Contact Settings* section.  Note: the Personal contact form is selected by default.

File: docs/user-documentation/users.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+1. Click on the **Create New Account** button at the bottom of the *Add user* page to finish adding a new user.
+
+To review/edit the permission for each role, in the *People* page click the **Permissions** tab in the set of tabs above the **Add user** button.
+
+## How to Create a New User Role
+1. Log in to Drupal under admin credentials.
```
Comment (librarychik) [inline docs/user-documentation/users.md]: Same as above:

Log in to Drupal.  You will need the Administrator role to edit or add roles.

File: docs/user-documentation/users.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+1. Click the **Manage** tab from the top toolbar.
+1. Select the **People** tab from the resulting menu.
+1. Click the **Roles** tab in the set of tabs above the **Add user** button.
+1. Click the blue **Add role** button to bring up the *Add role* page.
+![Alt text](../assets/users_people_roles.png "Roles page")
+1. Name the role in a way that it can be disambiguated from related activities or similar groups
+    * I.e. use the course code for a particular class of students working on a particular collection.
```
Comment (librarychik) [inline docs/user-documentation/users.md]: Add period after *similar groups*.  
Change I.e. to For example:

File: docs/user-documentation/users.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+1. Click the blue **Save** button.
+
+## How to Edit Role Permissions
+1. Log in to Drupal under admin credentials.
```
Comment (librarychik) [inline docs/user-documentation/users.md]: Same as above:
Log in to Drupal. You will need the Administrator role to edit permissions.

File: docs/user-documentation/users.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+1. Click the **Manage** tab from the top toolbar.
+1. Select the **People** tab from the resulting menu.
+1. Click the **Roles** tab in the set of tabs above the **Add user** button.
+![Alt text](../assets/users_people_roles.png "Roles page")
+1. To edit the permissions for a role click the **Edit** dropdown menu to the right of a role and select **Edit Permissions** to bring up the *Edit role* page.
+1. Scroll down or search for options that have an *Islandora* prefix or contain the word *Islanodra*. For example, *Islandora Access: Create terms*.
```
Comment (librarychik) [inline docs/user-documentation/users.md]: Spelling: 
_or contain the word **Islanodra**_  change to **Islandora**

File: docs/user-documentation/users.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+# Managing Users
```
Comment (Natkeeran) [inline docs/user-documentation/users.md]: Providing link to this documentation https://www.drupal.org/docs/user_guide/en/user-new-user.html would be useful.  Perhaps like:

Further Reading:
* [Drupal.org Managing User Acccounts](https://www.drupal.org/docs/user_guide/en/user-chapter.html)

**Comment by ysuarez** [comment]
Thank you @librarychik & @Natkeeran 

I will work to implementing these changes as soon as I can today or tomorrow.

**Comment by ysuarez** [comment]
@librarychik & @Natkeeran how does it look now?

**Comment by ysuarez** [comment]
@dannylamb can you give me some feedback on this part of my content quoted below (still don't know how to properly directly quote content from my commits)....

> 1. In the *Roles* section, click on the roles that the new user will need.
>  - Click on "Administrator" if the new user will need to be able to run administrative tasks in Drupal or Islandora 8.
>  - Click on "fedoraAdmin" if the new user will need to be able to run certain tasks in Fedora. This is required by the WebAC authentication so Islandora 8 and Fedora can communicate.

**Comment by dannylamb** [comment]
@ysuarez Circling back around to all the leftover sprint tickets after a long vacay.  Thanks for your patience on this one.

Instead of "run certain tasks in Fedora" I'd put "write to Fedora".  If you want to update that, we'll I'm more than happy to merge.

**Comment by ysuarez** [comment]
@dannylamb I just made the change you suggested. Look it over and merge if you approve.

**Pull request merged**